addons/account/models/account_journal_dashboard.py

```diff
@@ -54,7 +55,7 @@ def _get_json_activity_data(self):
                     'res_id': activity.get('res_id'),
                     'res_model': activity.get('res_model'),
                     'status': activity.get('status'),
-                    'name': activity.get('summary') or activity.get('act_type_name'),
+                    'name': (activity.get('summary') or activity.get('act_type_name')) + '(' + format_date(activity.get('date'), 'MMM', locale=self._context.get('lang') or 'en_US') + ')',
```

Comment:
What part of the spec does this correspond to?

Reply:
7th page in doc asking to add period in